### PR TITLE
feat(route/yicai): Add route for carousel part on front page

### PR DESCRIPTION
**Pull request by dzx-dzx**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/yicai/carousel
/yicai/news
/yicai/latest
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Missing pubDate
Wrong route maintainers

**Comment by dzx-dzx** [comment]
> Missing pubDate Wrong route maintainers

I don't undeerstand why, but the `pubDate` can be parsed on my machine.

File: lib/routes/yicai/utils.ts [hunk 1]
```diff
@@ -35,25 +35,27 @@ const ProcessItems = async (apiUrl, tryGet) => {
...
+                method: 'get',
+                url: item.link,
+            });
 
-                const content = load(detailResponse.data);
+            const content = load(detailResponse.data);
 
-                content('h1').remove();
-                content('.u-btn6, .m-smallshare, .topic-hot').remove();
+            item.pubDate = item.pubDate ?? timezone(parseDate(content('em.f-fl').text()), +8);
```
Comment (TonyRL) [inline lib/routes/yicai/utils.ts:50]: Please use the value of `actime` in `_pb` instead.

**Review by TonyRL — COMMENTED** [review]
```xml
<guid isPermaLink="false">https://www.yicai.com/news/102231917.html</guid>
      <pubDate>Invalid Date</pubDate>
    </item>
```

**Pull request merged**